File: Countries/United/united.py
# toy program to make a list of all the countries with "United" in the name

# importing the country list
countries = open("/home/evergreen/UnderDogsAssignments/Countries/countries.txt")

#turning countries.doc into an array
countries_list = []
for line in countries:
    countries_list.append(line.strip())

# go through countries_list and add all of the ones containing "United" to countries_united
countries_united = []
for country in countries_list:
    if "United" in country:
        countries_united.append(country)
        
print("The countries' who's names contain 'United' are " + str(countries_united))

File: Countries/United/united.js
// Program to create a list of all countries containing "United"

// Importing the fs module
let fs = require("fs")

// Intitializing the readFileLines with the file
const readFileLines = filename =>
   fs.readFileSync(filename)
   .toString('UTF8')
   .split('\n');

// Calling the readFiles function with file name
let countries = readFileLines('/home/evergreen/UnderDogsAssignments/Countries/countries.txt');

// const united = [];
// for (var i = 0; i < countries.length -1; i++) {
//    if (countries[i].includes("United")) {
//       united.push(countries[i]);
//    }
// }

// console.log(united)

const states = [];
for (var i = 0; i < countries.length -1; i++) {
   if (countries[i].includes("States")) {
      states.push(countries[i]);
   }
}

console.log(states)